rclpy/test/test_client.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+import unittest
+
+from rcl_interfaces.srv import GetParameters
+import rclpy
+
+
+# TODO(sloretz) Reduce fudge once wait_for_service uses node graph events
+TIME_FUDGE = 0.3
+
+
+class TestClient(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        rclpy.init()
+        cls.node = rclpy.create_node('TestClient')
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.node.destroy_node()
+        rclpy.shutdown()
+
+    def test_wait_for_service_5sec(self):
+        cli = self.node.create_client(GetParameters, 'get/parameters')
+        try:
+            start = time.monotonic()
+            cli.wait_for_service(timeout_sec=5.0)
```

Comment:
should the return value be checked ? (same below)
Also it would be great (likely another PR) to have tests for when the service does exist

Reply:
Check return value of wait for service 35f19af454a852b76f974896297c446cb9c30015

Add test where service exists 111a4cb6da8b682448950abf5e858205412cb7b6

Retriggered CI